src/screenControl.py

```diff
@@ -370,6 +381,8 @@ def processInput(self, key):
             # before exiting the program
             self.getFilesToUse()
             self.cursesAPI.exit()
+        elif self.mode == X_MODE and key in lbls:
```

Comment:
this might prone to breakage in the future since we might add keys above this statement that are in `lbls` and forget about `X_MODE`. i wonder if we could instead refactor some of this...